src/exchange/billing/account-renew/billing-account-renew.html

```diff
@@ -0,0 +1,136 @@
+                </oui-column>
+                <oui-column
+                    data-title="::'exchange_update_billing_header_1M' | translate">
+                    <oui-radio name="renewPeriod-{{::$rowIndex}}"
+                        data-ng-if="!$ctrl.account.partial && $ctrl.canHaveMonthlyRenewal()"
+                        data-model="$row.renewPeriod"
+                        data-on-change="$ctrl.trackSelected($row.primaryEmailAddress, $row.renewPeriod)"
+                        data-value="'MONTHLY'">
+                        {{::'exchange_update_billing_periode_value_MONTHLY' | translate}}
+                    </oui-radio>
+                </oui-column>
+                <oui-column
+                    data-title="::'exchange_update_billing_header_1Y' | translate">
+                    <oui-radio name="renewPeriod-{{::$rowIndex}}"
+                        data-ng-if="!$ctrl.account.partial"
+                        data-model="$row.renewPeriod"
+                        data-on-change="$ctrl.trackSelected($row.primaryEmailAddress, $row.renewPeriod)"
+                        data-value="'YEARLY'">
+                        {{::'exchange_update_billing_periode_value_YEARLY' | translate}}
+                    </oui-radio>
+                </oui-column>
+                <oui-column
+                    data-title="::'exchange_update_billing_header_terminate' | translate">
+                    <oui-radio name="renewPeriod-{{::$rowIndex}}"
+                        data-ng-if="!$ctrl.account.partial && $ctrl.canBeDeletedAtExpiration()"
+                        data-model="$row.renewPeriod"
+                        data-on-change="$ctrl.trackSelected($row.primaryEmailAddress, $row.renewPeriod)"
+                        data-value="'DELETE_AT_EXPIRATION'">
+                        {{::'exchange_update_billing_periode_value_DELETE_AT_EXPIRATION' | translate}}
+                    </oui-radio>
+                </oui-column>
+            </oui-datagrid>
+
+            <oui-message class="mb-5"
+                data-type="warning"
+                data-ng-if="$ctrl.canBeDeletedAtExpiration() && $ctrl.services.exchangeVersion.isAfter(2010, $ctrl.exchange) && !$ctrl.account.partial && $ctrl.model.displayDeleteWarning">
+                <span data-translate="exchange_update_billing_periode_delete_warning"></span>
+            </oui-message>
+        </oui-step-form>
+
+        <oui-step-form
+            data-header="{{::'exchange_update_billing_validation_title' | translate}}"
+            data-description="{{::'exchange_update_billing_periode_resume' | translate}}"
+            data-loading="$ctrl.submitLoader">
+            <div class="oui-datagrid-responsive-container" data-ng-if="!$ctrl.submitLoader">
+                <div class="oui-datagrid-responsive-container__sticky-container">
+                    <div class="oui-datagrid-responsive-container__overflow-container">
+                        <table class="oui-datagrid">
```

Comment:
Might be worth using a datagrid to avoid handling all the style ourselves. What do you think ?

Reply:
IMO, it's going out the tank for a simple grid. Even the responsive part is not necessary here.
But the pros for using it is that we standardize the use of the component instead of the classic table (that still works :p). Currently, is there a preferred way in the manager ?